depthai_nodes/ml/parsers/yunet.py

```diff
@@ -49,28 +53,24 @@ def __init__(
         @type max_det: int
         @param input_shape: Input shape of the model (width, height).
         @type input_shape: Tuple[int, int]
+        @param loc_output_layer_name: Output layer name for the location predictions.
+        @type loc_output_layer_name: str
+        @param conf_output_layer_name: Output layer name for the confidence predictions.
+        @type conf_output_layer_name: str
+        @param iou_output_layer_name: Output layer name for the IoU predictions.
+        @type iou_output_layer_name: str
         """
-        super().__init__()
-        self._out = self.createOutput(
-            possibleDatatypes=[
-                dai.Node.DatatypeHierarchy(dai.DatatypeEnum.ImgDetections, True)
-            ]
-        )
```

Comment:
The `self._out = self.createOutput` part was suggested to be added by [lnotspotl](https://github.com/lnotspotl) #90 to fix the connection between parsers. I'd keep it in!